app/models/user.rb

```diff
@@ -577,7 +577,7 @@ def validate_mastodon_url
   end
 
   def tag_keywords_for_search
-    employer_name.to_s + mostly_work_with.to_s + available_for.to_s
+    "#{employer_name}#{mostly_work_with}#{available_for}"
```

Comment:
Neiter @maestromac nor @benhalpern  really knew what this method is for. It's only used in `search_core`, which is then wrapped one more time in `hotness_score`.

```ruby
  def hotness_score
    search_score
  end

  def search_score
    counts_score = (articles_count + comments_count + reactions_count + badge_achievements_count) * 10
    score = (counts_score + tag_keywords_for_search.size) * reputation_modifier
    score.to_i
  end
```

Given that `tag_keywords_for_search` just concatenates 3 strings into one, I think the idea was to give new users (where `counts_score` will be 0) with filled in profiles a higher score than people with no profile information.